brave/browser-laptop
# Bat-client version bump

Fixes #13217
Fixes #13875

## Submitter Checklist:


## Test Plan:

1. enable payments
2. go to https://www.youtube.com/channel/UCd534c_ehOvrLVL2v7Nl61w or https://www.youtube.com/channel/UCtinbF-Q-fVthA0qrFQTgXQ
3. watch video on this site
4. make sure that there is no strange entries in ledger table

## Reviewer Checklist:


Tests
